src/foam/u2/view/TableCellFormatter.js

```diff
@@ -427,6 +427,33 @@ foam.CLASS({
 });
 
 
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'DateTimeUTCTableCellFormatterRefinement',
+  refines: 'foam.lang.DateTimeUTC',
+
+  properties: [
+    {
+      class: 'foam.u2.view.TableCellFormatter',
+      name: 'tableCellFormatter',
+      value: function(date) {
+        if ( date ) {
+          // Format as UTC using locale-default format
+          var formattedDate = foam.util.DateUtil.format(date, 'UTC');
+          this.add(formattedDate);
+          this.tooltip = formattedDate;
+        }
+      }
+    },
+    {
```

Comment:
Is this needed?

Reply:
the tablewidth? I saw it was defined for datetime, so wanted to make look the same

Reply:
removed. not needed